### ISSUE TITLE
Tibia 12 support

### ISSUE THREAD

**Issue by CalldiDoctor** [edubart/otclient#1079]
Not sure if this should be asked here, but I've seen there are other issues like this marked as **Missing feature**.

Are there any plans on adding support to the latest Tibia protocol? AFAIK, 12.30 currently. 
I could be interested in collaborating. 

Thanks!

**Comment by Tazer89** [comment]
> Not sure if this should be asked here, but I've seen there are other issues like this marked as **Missing feature**.
> 
> Are there any plans on adding support to the latest Tibia protocol? AFAIK, 12.30 currently.
> I could be interested in collaborating.
> 
> Thanks!

https://github.com/OTCv8/otclientv8

**Comment by CalldiDoctor** [comment]
oh, didn't know about that. Thanks :+1: 

Does it also support the latest version. I see it says _Full tibia 11.00 support_.

**Comment by Tazer89** [comment]
> oh, didn't know about that. Thanks 👍
> 
> Does it also support the latest version. I see it says _Full tibia 11.00 support_.

Yes it supports 12x but still have some features that not working still. But it will come soon i guess.
